tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -1606,30 +1641,14 @@ def test_failure_rate_alias_field(self):
         assert data[0]["failure_rate()"] == 0.75
 
     def test_count_miserable_alias_field(self):
-        events = [
-            ("one", 300),
-            ("one", 300),
-            ("two", 3000),
-            ("two", 3000),
-            ("three", 300),
-            ("three", 3000),
-        ]
```

Comment:
The events differ a bit with the setup function, thus, the response in data is slightly different.
<img width="346" alt="image" src="https://github.com/getsentry/sentry/assets/44410/ad554ef9-b762-48df-9946-6ff016d9e2b1">

<img width="248" alt="image" src="https://github.com/getsentry/sentry/assets/44410/270de2b9-0f84-4ded-b20b-de26b8b41673">